tutorials/arvo/arvo.md

```diff
@@ -4,6 +4,10 @@ weight = 1
 template = "doc.html"
 aliases = ["/docs/learn/arvo/arvo/"]
 +++
+
+Parts of this document are out of date, please use information here with
+caution. It is being updated as of 2020.07.20.
```

Comment:
Giving a date for when it is being updated (a future occurrence) is a little confusing. Maybe add a phrase saying that the message will be removed when it's updated?

Also using 

> This

formatting might make it stand out more.

Reply:
Fair, how's this sound

https://github.com/urbit/docs/pull/917/commits/613bc58ec6cb671abe776571026662dec2ffde4a